settings.gradle

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
     plugins {
-        id 'io.micronaut.build.shared.settings' version getProperty("micronautSharedSettingVersion")
```

Comment:
You can modify the version in gradle properties

Reply:
Yes, I understand. But we should keep the project's build align. Having the version in `settings.gradle`, gets automatic version upgrades PR.

If we want to define build tools version in `gradle.properties`  we should change it in project template and then use it in every project.